src/system/RoutesModule/Resources/config/services.yml

```diff
@@ -1,144 +1,110 @@
 parameters:
-    # Route parts
-    zikularoutesmodule.routing.ajax: ajax
-    zikularoutesmodule.routing.external: external
-    zikularoutesmodule.routing.view.suffix: view
-    zikularoutesmodule.routing.route.singular: route
-    zikularoutesmodule.routing.route.plural: routes
-    zikularoutesmodule.routing.formats.view: html|kml
-    zikularoutesmodule.routing.formats.display: html|kml|ics
-
-    # Entity factory classes
-    zikularoutesmodule.entity.factory.route.class: Zikula\RoutesModule\Entity\Factory\RouteFactory
-
-    # Listener classes
-    zikularoutesmodule.core_listener.class: Zikula\RoutesModule\Listener\CoreListener
-    zikularoutesmodule.kernel_listener.class: Zikula\RoutesModule\Listener\KernelListener
-    zikularoutesmodule.installer_listener.class: Zikula\RoutesModule\Listener\InstallerListener
-    zikularoutesmodule.moduledispatch_listener.class: Zikula\RoutesModule\Listener\ModuleDispatchListener
-    zikularoutesmodule.mailer_listener.class: Zikula\RoutesModule\Listener\MailerListener
-    zikularoutesmodule.page_listener.class: Zikula\RoutesModule\Listener\PageListener
-    zikularoutesmodule.theme_listener.class: Zikula\RoutesModule\Listener\ThemeListener
-    zikularoutesmodule.view_listener.class: Zikula\RoutesModule\Listener\ViewListener
-    zikularoutesmodule.userlogin_listener.class: Zikula\RoutesModule\Listener\UserLoginListener
-    zikularoutesmodule.userlogout_listener.class: Zikula\RoutesModule\Listener\UserLogoutListener
-    zikularoutesmodule.user_listener.class: Zikula\RoutesModule\Listener\UserListener
-    zikularoutesmodule.userregistration_listener.class: Zikula\RoutesModule\Listener\UserRegistrationListener
-    zikularoutesmodule.users_listener.class: Zikula\RoutesModule\Listener\UsersListener
-    zikularoutesmodule.group_listener.class: Zikula\RoutesModule\Listener\GroupListener
-
-    # Util classes
-    zikularoutesmodule.model_helper.class: Zikula\RoutesModule\Util\ModelUtil
-    zikularoutesmodule.controller_helper.class: Zikula\RoutesModule\Util\ControllerUtil
-    zikularoutesmodule.view_helper.class: Zikula\RoutesModule\Util\ViewUtil
-    zikularoutesmodule.workflow_helper.class: Zikula\RoutesModule\Util\WorkflowUtil
-    zikularoutesmodule.listentries_helper.class: Zikula\RoutesModule\Util\ListEntriesUtil
-
-    
-    # Log processor
-    zikularoutesmodule.log.processor.class: Monolog\Processor\PsrLogMessageProcessor
```

Comment:
Nevermind, the issue occurs due to the custom definitions in the `Resources/config/custom_services.xml` file.